cxx-sensors/src/main/java/org/sonar/cxx/sensors/pclint/CxxPCLintSensor.java

```diff
@@ -50,6 +50,8 @@
   private static final Logger LOG = Loggers.get(CxxPCLintSensor.class);
   public static final String REPORT_PATH_KEY = "pclint.reportPath";
   public static final String KEY = "PC-Lint";
+  public static final Boolean MISRA = false;
```

Comment:
@MarcAntoineCRUE these flags are confusing. Why not just set true / false in the function?